### ISSUE TITLE
iOS setup

### ISSUE THREAD

**Issue by kevinsmithwebdev** [mouselangelo/react-native-actions-shortcuts#8]
I'm still having difficulty getting iOS to work, but one speedbump I had has an error as I was trying to build to the app, that it wasn't understanding "RNShortcuts" in AppDelegate.m. When I added:
```
#import "RNShortcuts.h"
```
to the top of that file, it now at least builds. I don't know if that was an oversight, something that is just assumed, or if I'm way off base. I don't know - I don't do objective-C so I had to figure it out.

**Comment by Dipeshwagle** [comment]
Yes this is required, It's included in the example app's `AppDelegate.m`.
